masterlist.yaml

```diff
@@ -2501,16 +2530,49 @@ plugins:
   - name: 'AOS.esp'
     after:
       - 'WetandCold.esp'
+      # RWT must be loaded after Requiem but AOS shouldn't so this rule is disabled when Requiem is installed
```

Comment:
But this rule (conditionally) says to load `AOS.esp` `after` `RealisticWaterTwo.esp`, not after Requiem... ? Does RWT load after Requiem or did you make a typo or... ?